### PR TITLE
Daniel

### LINKED ISSUE #24: Create route that sends prompt to Gemini API

### LINKED ISSUE #25: Create Helper function for the Gemini route

### LINKED ISSUE #26: Testing for GeminiRouter

### PR DESCRIPTION
**Pull request by danxguerrero**
closes #24, closes #25, closes #26

### CODE REVIEW

File: src/services/gemini/GeminiService.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+// Access the GEMINI API key as an environment variable
+const genAI = new GoogleGenerativeAI(process.env.GEMINI_API_KEY);
+
+
+async function runGemini(focus_area, type, level) {
+    const model = genAI.getGenerativeModel({ model: "gemini-1.5-flash"})
+
+    const prompt = `Please Generate a ${focus_area}, ${type} workout that for a ${level}. Please do not include any extra text or important notes. Do not put the JSON in a block quore. The workout should be in the following JSON format: 
+      {
```
Comment (Aaron-G-Sanchez) [inline src/services/gemini/GeminiService.js:12]: Can this be written like the [Example](https://ai.google.dev/gemini-api/docs/json-mode?authuser=1&lang=node#supply-schema-in-prompt)?

**Review by Aaron-G-Sanchez — CHANGES_REQUESTED** [review]

File: src/app.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ const jwtCheck = auth({
 
 app.use(cors())
 app.use(morgan('dev'))
-app.use(jwtCheck)
+// app.use(jwtCheck)
```
[the remark this answers is not in the document]
Comment (danxguerrero) [reply src/app.js:21]: Done

File: src/routes/gemini/GeminiRouter.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+const express = require('express')
+require('dotenv').config()
+const { runGemini } = require('../../services/gemini/GeminiService')
+
+const GeminiRouter = express.Router()
```
[the remark this answers is not in the document]
Comment (danxguerrero) [reply src/routes/gemini/GeminiRouter.js:5]: Done.

File: src/services/gemini/GeminiService.js (hunk 1 above)
Comment (danxguerrero) [reply src/services/gemini/GeminiService.js:12]: Done.

File: src/routes/index.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 const UserRouter = require('./users/UserRouter')
+const { GeminiRouter } = require('./gemini/GeminiRouter')
```
[the remark this answers is not in the document]
Comment (danxguerrero) [reply src/routes/index.js:2]: Done.

**Review by Aaron-G-Sanchez — APPROVED** [review]
LGTM!

**Pull request merged**